index.html

```diff
@@ -209,7 +209,10 @@ <h3 id="string_specific_language">String-specific language information</h3>
 
 <p class="advisement" id="bp-lang_field_based_metadata"><a class="self" href="#bp-lang_field_based_metadata">&#x200B;</a>Use field-based metadata or string datatypes to indicate the language and the base direction for individual natural language strings.</p>
 
-<p> Low-level support for natural language string metadata is widespread because the use of metadata for storage and interchange of the language of data values is long-established and widely supported in the basic infrastructure of the Web. This includes language attributes in [[XML]] and [[HTML]]; string types in schema languages (e.g. [[xmlschema11-2]]) or the various RDF specifications including [[JSON-LD]]; or protocol- or document format-specific provisions for language.</p>
+<p>Low-level support for natural language string metadata is widespread because the use of metadata for storage and interchange of the language of data values is long-established and widely supported in the basic infrastructure of the Web. This includes language attributes in [[XML]] and [[HTML]]; string types in schema languages (e.g. [[xmlschema11-2]]) or the various RDF specifications including [[JSON-LD]]; or protocol- or document format-specific provisions for language.</p>
+
+<p class="advisement" id="bp-use_jsonld_i18n_namespace"><a class="self" href="#bp-use_jsonld_i18n_namespace">&#x200B;</a>Specify the use of [[JSON-LD]] 1.1 <code>i18n namespace</code> feature for RDF literals.</p>
+
```

Comment:
This is not a "JSON-LD 1.1 i18n namespace". It is a generic namespace defined for RDF which has been defined in the JSON-LD document (for all the known reasons) but can be used in any RDF serialization _including_ JSON-LD. Although I know this is not the subject of this document, but the same properties could be used in Turtle, for example.

Maybe:

> Specify the use of the <code>i18n namespace</code> feature for RDF literals, as defined in the [[JSON-LD]] 1.1 document

Reply:
Thanks. I adopted this text.